Analyzers/Microsoft.Identity.VersionMatchAnalyzer/Microsoft.Identity.VersionMatchAnalyzer.Package/Microsoft.Identity.VersionMatchAnalyzer.Package.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
can we also ship net472?

Reply:
No, my understanding was netstandard2.0 can work for net472 projects also.